var/spack/repos/builtin/packages/ascent/package.py

```diff
@@ -189,6 +193,7 @@ class Ascent(CMakePackage, CudaPackage):
     depends_on("raja~openmp", when="+raja ~openmp")
     depends_on("umpire", when="+umpire")
     depends_on("umpire@:2023.06.0", when="@:0.9.2 +umpire")
+    depends_on("umpire@2024.02:", when="@0.9.3: +umpire")
```

Comment:
It sounds like @cyrush is saying a similar constraint would make sense for `raja`. @white238 would you prefer to do that here? Based on checking the pipeline it looks like we got build coverage over your new version, but the constraint could help avoid problems in other contexts.